### PR TITLE
Added mobile menu to the mobile version of the  Portfolio-Project

### PR DESCRIPTION
**Pull request by SaeqaSultani**
- Added a .gitignored file.
- Set up a linter for CSS ,HTML  and JavaScript.
- Added a readme file.
- Added a .html file.
- Added a .css file.
- Added a . js file.
- Added a test.md file
### instruction 
When the user clicks (or taps) the hamburger button, the mobile menu appears.
When the user clicks (or taps) the close (X) button, the mobile menu disappears.
When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+const close = () => {
+  menulist.classList.remove('internal-link-mobActive');
+  main.style.opacity = '1';
+};
+
+icon.addEventListener('click', open);
+menu.forEach((menuLink) => {
+  menuLink.addEventListener('click', close);
+});
```
Comment (zhadier) [inline main.js]: - [ ] Great job implementing the functionality for the hamburger. But kindly refer to the images below and increase the blur effect in the background so it resembles the Figma template. As you can see the Figma mobile menu has a much darker background with a less visible backdrop.

> Figma Design

image

> Your Design

image

- [ ] Additionally, kindly also implement the black horizontal line at the end of the Mobile menu like I've highlighted in the Figma template above.

**Review by zhadier — CHANGES_REQUESTED** [review]
# Hi @SaeqaSultani  👋 

Good job so far! 🌤️ 
There are some issues that you still need to work on to go to the next project but you are almost there! 🍓 



## Highlights
✔️ No linter errors
✔️ Github flow was set up correctly
✔️ Professional pull request title


## Required Changes ♻️

- [ ] In your PR descriptions, kindly also mention the features you added so we know what exactly has been implemented in the given branch

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question @zhadier so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -22,6 +22,85 @@
...
+  margin-top: 12px;
+  margin-right: 20px;
+}
+
+.internal-link-mobActive {
+  width: 100%;
+  margin: 0;
+  padding-left: 40px !important;
+  background-color: rgb(16, 37, 222) !important;
+}
```
Comment (vechicin) [inline style.css]: While you did implement a previous reviewer's requirement, you're still missing some other ones that were requested.

Screen Shot 2022-07-27 at 7 17 48

Screen Shot 2022-07-27 at 7 18 10

The background is not blurred yet, and the black divider at the bottom doesn't have the correct color nor the correct positioning. Please fix them.

Additionally, I am now experiencing a horizontal scroll that makes for bad UI/UX.

Screen Shot 2022-07-27 at 7 22 33

Please fix it as well.

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @SaeqaSultani ✋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 🚨 


 ## Highlights ⚡ 

- [x] Good job implementing some of the previous reviewer's requirements

 ## Required Changes ♻️

- [ ] You still need to tweak your PR description. It is not enough to list the files you added for this project, you need to verbally describe the features you implemented on this branch. Please do so.

_Check the comments under the review._

## Optional suggestions 📂 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by SaeqaSultani** [comment]
Please kindly explain to me what is your mind about the PR description
because this is my last chance.

File: index.html [hunk 3]
```diff
@@ -13,43 +13,63 @@
...
-                <li><a href="#">About</a></li>
-                <li><a href="#"> Contact </a></li>
-             </ul>
-          </nav>
+            <nav class="nav-mob">
+                <a class="title">My logo</a>
+                <a class="icon-menu" href="#"><img src="images/Union.png" alt="my logo" /></a>
+            </nav>
+
+            <ul class="internal-link-mob">
```
Comment (Ismailco) [inline index.html:21]: - [ ] [OPTIONAL] Good job implementing the requested changes, you can give the menu element's some space or remove the margin so they are closer to the edge of the page like in the Figma design

Screenshot_20220727_210418

**Review by Ismailco — APPROVED** [review]
### Hi @SaeqaSultani, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by SaeqaSultani** [comment]
weekly Meet
Tuesday Sep 27, 2022



Organizer
***@***.***
***@***.***

Guests
***@***.*** - organizer
***@***.***
View all guest info  
https://calendar.google.com/calendar/event?action=VIEW&eid=M3IwN2NjaHE4NmVva2pkajNnODZkdjlrdGFfMjAyMjA5MjcgcmVwbHkrYXIyeGs1bXA1YWJ2bGY1dmdnaXQ1ZzZhNXpxc3ZldmJuaGhlNWlvYzV1QHJlcGx5LmdpdGh1Yi5jb20&tok=MjMjc2FlcWEuc3VsdGFuaUBnbWFpbC5jb20zYzNkMmUxMDMyMWFhZjViNWUzMDFmMmEyOWUxNGIyMmRiYmQyYmU0&ctz=Asia%2FKabul&hl=en&es=0

Reply for ***@***.***  
and view more details  
https://calendar.google.com/calendar/event?action=VIEW&eid=M3IwN2NjaHE4NmVva2pkajNnODZkdjlrdGFfMjAyMjA5MjcgcmVwbHkrYXIyeGs1bXA1YWJ2bGY1dmdnaXQ1ZzZhNXpxc3ZldmJuaGhlNWlvYzV1QHJlcGx5LmdpdGh1Yi5jb20&tok=MjMjc2FlcWEuc3VsdGFuaUBnbWFpbC5jb20zYzNkMmUxMDMyMWFhZjViNWUzMDFmMmEyOWUxNGIyMmRiYmQyYmU0&ctz=Asia%2FKabul&hl=en&es=0
Your attendance is optional.

~~//~~
Invitation from Google Calendar: https://calendar.google.com/calendar/

You are receiving this email because you are an attendee on the event. To  
stop receiving future updates for this event, decline this event.

Forwarding this invitation could allow any recipient to send a response to  
the organizer, be added to the guest list, invite others regardless of  
their own invitation status, or modify your RSVP.

Learn more https://support.google.com/calendar/answer/37135#forwarding

**Pull request merged**